pages.id/common/gh.md

```diff
@@ -0,0 +1,37 @@
+# gh
+
+> Memudahkan pengaksesan GitHub dari command-line.
+> Beberapa subcommands seperti `gh config` memiliki dokumentasi sendiri.
+> Informasi lebih lanjut: <https://cli.github.com/>.
+
+- Mengklon sebuah GitHub repositori di lokal
+
+`gh repo clone {{pemilik}}/{{repositori}}`
+
+- Membuat issue baru:
```

Comment:
We have the translation for `issue` that doesn't change familiarity / meaning, which is `isu`.